internal/olm/client.go

```diff
@@ -0,0 +1,467 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package olm
```

Comment:
Do we want to at some point move these in OLM project itself and use it as a library instead? I think we mentioned this in the proposal. If yes, can we add a TODO to remember moving this.

Reply:
I'm not sure exactly what we want to do, but we should at least talk more with the OLM team about it. I'll add the TODO as a reminder.